site/docs/2.0/content/font.md

```diff
@@ -13,57 +13,51 @@ toc: true
   your project.
 </div>
 
-## Centrally managed Typekit webfont
+## How to Use
+The Proxima Nova font suite is available for official use by University of Arizona
+employees through a license with Adobe Typekit.
 
-For ease of integration into web projects around campus, the Arizona Digital
-team manages a Typekit webfont project that can be referenced by your site.
+### Reference link
 
-#### Notes to consider when using the centrally mananged webfont project.
-Our license with Typekit allows anyone with a netid to create their own webfont
-project in Creative Cloud Typekit and use it in all of their web projects. Of
-course there are advantages and disadvantages to each approach.
+Arizona Bootstrap uses a Creative Cloud Library with shared governance to
```

Comment:
Is this accurate?  I thought we abandoned the idea of using Adobe's Creative Cloud Library feature and are instead just using straight-up typekit with a centrally managed shared login now...

Reply:
Yes, the Creative Cloud Library feature doesn't work.

Reply:
Does this description work?